### ISSUE TITLE
Can this model be trained on bert large?

### ISSUE THREAD

**Issue by Umang9427** [kamalkraj/BERT-NER#74]

**Comment by ranjeetds** [comment]
Yes. While running ```run_ner.py``` script change ```--bert_model``` argument to ```--bert_model=bert-large-cased/uncased```

**Comment by Umang9427** [comment]
Yeah, but it will require higher specs of CUDA in machine, even google colab is not enough.

**Comment by ranjeetds** [comment]
You can always run it with a smaller batch size. You need to set ```--train_batch_size``` argument while running the script. For example, In my custom data case, some of the sentences are very long and hence I have to set ```max_seq_length``` to 512. I am using RTX 2080Ti GPU and with a batch size of 2 I can finish one epoch in ~4 hours (For ~60K training samples) for large bert model

**Comment by Swty13** [comment]
@ranjeetds 

Hi ,
I am training my custom NER model using BERT with 1 lakh dataset.Could you plz help me what hardware configuration is required to train this.

( I have VM having 32GB and 64GB RAM would it be possible to train on this or any GPU configuration is must to train this.)
